content/tutorial/02-sveltekit/02-routing/01-pages/README.md

```diff
@@ -4,13 +4,28 @@ title: Pages
 
 SvelteKit uses filesystem-based routing, which means that the _routes_ of your app — in other words, what the app should do when a user navigates to a particular URL — are defined by the directories in your codebase.
 
-In this app we have two routes — `src/routes/+page.svelte`, which maps to `/`, and `src/routes/about/+page.svelte`, which maps to `/about`. Clicking the `about` link will take you from the home page to the about page.
+The routes are located within `src/routes`. Every directory within which contains a `+page.svelte` file creates a route in your app.
 
-> Unlike traditional multi-page apps, navigating to `/about` updates the contents of the current page, like a single-page app. This gives us the best of both worlds — fast server-rendered startup, then instant navigation.
+In this app we currently have one route — `src/routes/+page.svelte`, which maps to `/`.
 
-Let's add a link in `src/routes/about/+page.svelte` back to the homepage:
+Let's add a second route, `src/routes/about/+page.svelte`, which maps to `/about`:
+
+```diff
+src/routes/
++├ about/
++│ └ +page.svelte
+└ +page.svelte
+```
+
+Clicking the `about` link on the home page will now take you to the about page (which is empty right now).
+
+Let's add some content (including a link back to the homepage) to `src/routes/about/+page.svelte` :
 
 ```svelte
++++<h1>About</h1>
```

Comment:
I got a popup when I tried to copy/paste this, but checking the box didn't seem to enable it. I would find it super annoying if I had to type all this and just gave up and hit solve

Reply:
mhm that's strange, I definetely checked this box multiple times already 😄

Reply:
You have to copy again after enabling it. Maybe we can write the content to the clipboard as soon as it's enabled

Reply:
Ah, yeah, I might not have realized that. Writing to the clipboard sounds nice

Reply:
Unfortunately this isn't possible — we can't programmatically read and write to and from the clipboard unless the user has given permission, and we can't request permission and _also_ call `event.preventDefault()` because permissions are async.